snuba/cli/multistorage_consumer.py

```diff
@@ -79,6 +80,11 @@
     "--output-block-size", type=int,
 )
 @click.option("--log-level")
+@click.option(
```

Comment:
Why not have this be something set by runtime config? That way we don't need ops to change something for us if we want to tweak this

Reply:
I think @nikhars at first didn't do it that way so that we didn't have to enable it for all consumers just yet

Reply:
We could set it per-consumer though. If the key is not set, don't turn it on?

Reply:
but we can think about using the runtime config and maybe using the consumer group appended to the value to do the look up